util/cf_options.cc

```diff
@@ -75,18 +75,14 @@ ImmutableCFOptions::ImmutableCFOptions(const ImmutableDBOptions& db_options,
       max_subcompactions(db_options.max_subcompactions) {}
 
 // Multiple two operands. If they overflow, return op1.
```

Comment:
I just noticed this comment which doesn't line up with what I assumed the function did on overflow.